### PR TITLE
Add Upsolver documentation to DBT docs

### PR DESCRIPTION
**Pull request by navado**
## What are you changing in this pull request and why?
Adding documentation for [dbt-upsolver](https://github.com/Upsolver/dbt-upsolver)

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Squash commits of this PR when merging

Adding new pages (delete if not applicable):
- [x] Add page to `website/sidebars.js`
- [x] Provide a unique filename for the new page

### CODE REVIEW

**Comment by navado** [comment]
@mirnawong1 @tommorris @dataders Can anyone please suggest how to proceed towards merging this documentation?

**Comment by dataders** [comment]
@navado thanks for opening this up! one note is that we're not currently accepting contributions for quickstarts, so if you can drop that, we can get this merged! hopefully the quickstart can have a home on y'all's docs site.

**Comment by navado** [comment]
@dataders Thanks for reply.
I removed the Quick Start Guide from this PR. Is it possible to merge?

File: website/docs/reference/resource-configs/upsolver-configs.md [hunk 1]
```diff
@@ -0,0 +1,413 @@
+---
+title: "Upsolver configurations"
+id: "upsolver-configs"
+description: "Upsolver Configurations - Read this in-depth guide to learn about configurations in dbt."
+---
+
+## Supported Upsolver SQLake functionality:
+
+| COMMAND | STATE | MATERIALIZED |
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/upsolver-configs.md]:
```suggestion
| Command | State | Materialized |
```

**Review by mirnawong1 — APPROVED** [review]
thansk for opening this up and for your patience @navado  , this looks great and happy to get this merged for you!

**Pull request merged**